### DISCUSSION TITLE
Mirror gRPC traffic

### DISCUSSION THREAD

**Discussion by JoAbyssinia** [istio/istio#48348] [Q&A] [answered]
I have a service that uses the gPRC protocol to communicate, I want to **mirror** the incoming request to a canary instance in the same cluster but I can't mirror it. 
I am sure I don't have an issue with the config but I don't know why I didn't see mirror traffic. 

is there any specific version of istio that has this future? 
currently, I am using `networking.istio.io/v1alpha3` version
 
thanks

**Comment by rootsongjc** [comment]
Are you sure the sidecar was injected into the pod?

**Reply by JoAbyssinia** [reply]
yes, everything is okay,

**Comment by JoAbyssinia** [comment, marked as answer]
I used -mesh in virtualservice gateway and solved it,
thanks
